src/config.h

```diff
@@ -23,6 +23,8 @@ enum {
     kTypeMuxDriver,           // 13 Multiplexer selector support (generates select outputs)
     kTypeDigInMux,            // 14 Digital input multiplexer support (example: 74HCT4067, 74HCT4051)
     kTypeStepper              // new stepper type with settings for backlash and deactivate output
+    // if new device types are added, change 'kTypeStepper' for the array numberDevices[] in readConfig() to the new one!!
```

Comment:
Suggestion: add a last entry like this (comments removed for clarity)
```C++
    // ...
    kTypeDigInMux,
    kTypeStepper,
    kTypeMAXCOUNT
```
and use that value for the array size in `getArraysizes()`. This way no manual update will be required when adding devices.

Reply:
Again a very good idea!